### ISSUE TITLE
No tokens on update 2.8

### ISSUE THREAD

**Issue by Chrishoee** [CityOfZion/neon-wallet#1473]
Wallet 0.2.8

Went to manage settings tokens and only have the locked script hashes but my tokens still do not show, please help.

Thank you

**Comment by topedaman** [comment]
same thing here but look up my public address thru neotracker all my tokens are there.

**Comment by tradebully** [comment]
Same here. It sucks I haven't been able to transfer my coins for three days now. C'mon guys this is taking too long

**Comment by madpug3** [comment]
I have the same problem. I moved my NEX tokens to my Neon Wallet. There is also GAS and Neo in there. All was fine for a while but now showing as “-“ for block number and also for balance of neo and gas and no tokens showing. I use a nano ledger S but this isn’t the problem as creating a new wallet (not through the ledger) has the same problem displaying “-“.

The tokens and coins show up fine on the public neotracker ledger so it appears to be neon wallet display issue but it is preventing me using any of my coins. Please fix this.

**Comment by comountainclimber** [comment]
@Chrishoee  @topedaman @tradebully  @madpug3  We are actively investigating this issue and seems related to some Neo 2.9.0 RPC nodes not responding to the getTokenBalances query... The simple temporary workaround is to to ping a different node - you can do this by toggling between main and test nets. Our next release will include manual node selection sorry for any inconvenience this may have caused

**Comment by madpug3** [comment]
Hi Comountainclimber, thanks for getting back to me so quickly. Unfortunately toggling between testnet and MainNet didn’t work and I still have the same problem. I have tried this with both the options of Neotracker and NeoScan selected. I appreciate you are actively trying to resolve the problem so I will just hang tight until the next release hoping that will fix it. Thanks.

**Comment by instafan** [comment]
> @Chrishoee @topedaman @tradebully @madpug3 We are actively investigating this issue and seems related to some Neo 2.9.0 RPC nodes not responding to the getTokenBalances query... The simple temporary workaround is to to ping a different node - you can do this by toggling between main and test nets. Our next release will include manual node selection sorry for any inconvenience this may have caused

Hey @comountainclimber any update with the fix?

**Comment by Gurustu** [comment]
Have same issue with 2.8  update.  I have not been able to use the waĺlet for about 1 month to access my Gas.  When will 2.9 be coming out and will it solve this issue?

**Comment by neilfarol** [comment]
I have the same issue as well. NEOscan shows all of my tokens but none on the NEON wallet.

**Comment by Kscramer** [comment]
Please provide status on fix to NEON wallet display.  I need to do some token transfers ASAP!

**Comment by instafan** [comment]
Just checked the other threads - did anyone try downgrading to 2.7 or 2.6? Or doing a clean uninstall / reinstall? Or trying a different wallet?

**Comment by Gurustu** [comment]
No, I have not tried these.  Might try downgrading first; I don't know how to uninstall/reinstall, will have to look that up. 


Best regards
 
Stuart Roberts
 
Telephone: 01580 240 191
Mobile: 07880 755 774
e-mail: robertsnutrition@aol.com
 



-----Original Message-----
